ppml/trusted-big-data-ml/scala/docker-occlum/kubernetes/executor_spark-logisticregressionexample.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
```

Comment:
We can re-used existing executor.yaml

Reply:
Yes. I use command `sed` to replace `container name` in `executor.yaml` in run_.sh in this pr.